changelog/pr786.json

```diff
@@ -0,0 +1,17 @@
+{
+    "pr_number": 786,
+    "changes": [
+        {
+            "change": "Features",
+            "description": "Added new archive/unarchive endpoints for posts, messages and users in order to archive and unarchive them. This allows the platform to be more mindful when 'deleting' items, offering recovery options as well as preparing a more managed way of permanently deleting items in future."
```

Comment:
"Allows the platform"? "A more managed way to deleting items in future"? I'm a little confused about what these mean